Sources/Client/Views/Shared/StyledTextfield.swift

```diff
@@ -1,8 +1,6 @@
 import SwiftUI
 
-
 struct StyledTextfield: View {
```

Comment:
Rename to `StyledTextField` to be in line with SwiftUI's spelling of `TextField`